hydra_agent/querying_mechanism.py

```diff
@@ -192,6 +192,19 @@ def get_members(self, query):
             print(self.connection.smembers("fs:endpoints"))
             return self.data_from_server(endpoint)
 
+    def insert_server(self, endpoint, new_object):
```

Comment:
What's the point of this function? You can directly call `insert_server` where you're calling this function.